### ISSUE TITLE
Local file support

### ISSUE THREAD

**Issue by teresam856** [GMOD/jbrowse-jupyter#5]
It would be awesome to have a way to support a user passing local files to use in the browser.

Example API: `config.set_assembly(data="path/to/my/cool.fa")`

Possible implementation:

- Check data, if it's not a URL, start Flask (silently under the hood)
- Use Flask to serve the data as a local URL

Some mechanics:

- [range requests with Flask](https://gist.github.com/lizhiwei/7885108)
- [Gradio](https://github.com/gradio-app/gradio/blob/master/gradio/networking.py) has some code for starting up Flask to serve stuff, but in our case we are serving the data, not the UI.

https://github.com/GMOD/dash_jbrowse/issues/5

**Comment by teresam856** [comment]
I created this file in the root dir of my computer to run an http server to serve my local files. Credit goes to: https://stackoverflow.com/questions/21956683/enable-access-control-on-simple-http-server

test_server.py
```
#!/usr/bin/env python3
from http.server import HTTPServer, SimpleHTTPRequestHandler, test
import sys

class CORSRequestHandler (SimpleHTTPRequestHandler):
    def end_headers (self):
        self.send_header('Access-Control-Allow-Origin', '*')
        SimpleHTTPRequestHandler.end_headers(self)

if __name__ == '__main__':
    test(CORSRequestHandler, HTTPServer, port=8000)
```
```$ python test_server.py```

While that server is running...

Replace the path `"http://[::]:8000/<path/to/your/genome>"` with the path to the file of your genome
browser.py
```
from dash import html, Dash
from jbrowse_jupyter import create, create_component
app = Dash(__name__)

conf = create("config")
new_assembly = conf.set_assembly("http://[::]:8000/<path/to/your/genome>", overwrite = True)

# Note: set_assembly assumes there will be a .fai file in the same directory unless you pass a specific path
# create a dash component
component = create_component(hg38.get_config(), dash_comp="LGV")

# launch the component
app.layout = html.Div(
    [component],
    id='test'
)

if __name__ == "__main__":
    app.run_server(port=3001, debug=True)

```
```$ python browser.py```

**Comment by teresam856** [comment]
Addressed in release v1.3.6.
